ui/src/app/components/dataflow-form/dataflow-form.component.ts

```diff
@@ -17,8 +17,11 @@ export class DataflowFormComponent implements OnInit {
     private dialofRef: MatDialogRef<DataflowFormComponent>
   ) {
     this.dataflowForm = new FormGroup({
-      network: new FormControl('', Validators.required),
-      subnetwork: new FormControl('', Validators.required),
+      network: new FormControl(''),
+      subnetwork: new FormControl(''),
```

Comment:
since, we have removed the required validators will it work in all cases where either network or sub network is specified and the other is not?

Reply:
There is still a backend check which will fail if only 1 is specified. However, we eventually need ot remove that check as well because network and subnetwork dont need to be specified together